### PR TITLE
Add --heap-stats for memory diagnostics

### PR DESCRIPTION
**Pull request by bbnickell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
   "type": "module",
   "scripts": {
     "start:indexer": "node indexer/index.js",
+    "start:indexer:heapstats": "node indexer/index.js --heap-stats",
     "start:indexer:prod": "pm2 start yarn --interpreter bash --name nina-indexer -- start:indexer",
+    "start:indexer:prod:heapstats": "pm2 start yarn --interpreter bash --name nina-indexer -- start:indexer --heap-stats",
```
Comment (facing-n) [inline package.json:11]: this line could use the `start:indexer:heapstats` script defined above, so:
`"start:indexer:prod:heapstats": "pm2 start yarn --interpreter bash --name nina-indexer -- start:indexer:heapstats"`

File: indexer/index.js [hunk 2]
```diff
@@ -20,6 +24,17 @@ const startProcessing = async () => {
 
   cron.schedule('* * * * *', async() => {
     console.log('Cron job starting: Sync Hubs + Releases');
+    if(arg[2]=="--heap-stats") {
+      console.log("Memory Diagnostics at " + new Date(Date.now()) + ": ");
```
Comment (facing-n) [inline indexer/index.js:28]: can these console logs go into a function, maybe `runHeapDiagnostics` so that we can keep the cron job clean?

**Review by facing-n — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (bbnickell) [reply package.json]: Added.

File: indexer/index.js (hunk 2 above)
Comment (bbnickell) [reply indexer/index.js]: Done.

**Review by facing-n — APPROVED** [review]

**Pull request merged**